docs/README.md

```diff
@@ -188,7 +197,7 @@ spec:
 
 </details>
 
-Note: More predefined policies and auto-discovered policy can be found here: [https://github.com/kubearmor/openhorizon-demo/tree/main/Open-Horizon/policies](https://github.com/kubearmor/openhorizon-demo/tree/main/Open-Horizon/policies)
+Note: More predefined policies and auto-discovered policy can be found here: [https://github.com/kubearmor/openhorizon-demo/tree/main/Open-Horizon/policies ](https://github.com/kubearmor/openhorizon-demo/tree/main/Open-Horizon/policies){:target="_blank"}{: .externalLink}
```

Comment:
> [https://github.com/kubearmor/openhorizon-demo/tree/main/Open-Horizon/policies ]

[https://github.com/kubearmor/openhorizon-demo/tree/main/Open-Horizon/policies]

Reply:
The extra space is intentional. It will separate the linked text from the launch glyph.